
The doc_dir returned by get_doc_dir() unfortunately contains both
the configured install_prefix (/usr) and DOC_DIR (/usr/share/doc),
resulting in a broken local URL file:///usr/usr/share/doc/..
Removing the install_prefix fixes the Help->Release notes menu.

Bug: https://bugs.gentoo.org/939195

--- a/wsutil/filesystem.c~	2024-09-06 21:33:04.000000000 +0200
+++ b/wsutil/filesystem.c	2024-09-06 21:33:30.900122945 +0200
@@ -1159,7 +1159,7 @@ get_doc_dir(void)
          */
         doc_dir = g_strdup(progfile_dir);
     } else {
-        doc_dir = g_build_filename(install_prefix, DOC_DIR, (char *)NULL);
+        doc_dir = g_build_filename(DOC_DIR, (char *)NULL);
     }
 #endif
     return doc_dir;
